app/navigation/Navigator.scala

```diff
@@ -46,6 +48,14 @@ class Navigator @Inject()() extends BaseNavigator {
     case CheckYourAnswersPage => _ =>
       //TODO: Route to the Confirmation Page as part of future story
       testOnly.controllers.routes.UnderConstructionController.onPageLoad()
```

Comment:
This needs to change to route to your new CancelConfirm page. e.g.:
```suggestion
    case CheckYourAnswersPage => _ =>
      routes.CancelConfirmController.onPageLoad(testErn, testArc, NormalMode)
```